inst/qml/SummaryStatsABTestBayesian.qml

```diff
@@ -30,15 +30,15 @@ Form
 	Group
 	{
 		title: qsTr("Group 1")
-		IntegerField { name: "y1";	label: qsTr("Successes")	}
-		IntegerField { name: "n1";	label: qsTr("Sample Size")	}
+		IntegerField { name: "y1";	label: qsTr("Successes"); max = n1.value}
```

Comment:
```suggestion
		IntegerField { name: "y1";	label: qsTr("Successes"); max: n1.value}
```
QML uses `:` and not `=` :wink:

Reply:
haha, thanks. Those are exactly the things that happen when you can't install the module to test the code....